TRACONBoundaries.geojson

```diff
@@ -145,6 +145,12 @@
     {"type":"Feature","properties":{"id":"ENTC","prefix":["ENTC"],"suffix":"APP","name":"Tromsø Approach"},"geometry":{"type":"MultiPolygon","coordinates":[[[[19.418055555555558,69.16027777777778],[18.26527777777778,69.36999999999999],[17.583333333333332,69.41666666666667],[17.333333333333332,69.66666666666667],[18.75,70.25],[19.25,70.25],[20,70.16250000000001],[20,69.45],[19.8275,69.36527777777778],[19.418055555555558,69.16027777777778]]]]}},
     {"type":"Feature","properties":{"id":"ENVA","prefix":["ENVA"],"suffix":"APP","name":"Værnes Approach"},"geometry":{"type":"MultiPolygon","coordinates":[[[[10.855555555555556,64.1],[11.995833333333332,63.94305555555555],[12.212913888888888,63.47856166666667],[11.974444444444444,63.26916666666666],[11.980182222222222,63.25],[11.75,63],[10.133333333333335,63],[9.8725,63.24583333333334],[9.809166666666668,63.48166666666667],[10.044444444444444,63.68055555555555],[10.14861111111111,63.71666666666667],[10.280277777777778,63.76138888888889],[10.408055555555556,63.80472222222222],[10.493055555555555,63.833333333333336],[10.855555555555556,64.1]]]]}},
     {"type":"Feature","properties":{"id":"ENZV","prefix":["ENZV"],"suffix":"APP","name":"Sola Approach"},"geometry":{"type":"MultiPolygon","coordinates":[[[[4.4991666666666665,59.60638888888889],[6.206666666666667,59.68527777777778],[6.721111111111111,58.73555555555556],[6.321111111111111,58.367222222222225],[5.199722222222222,58.35444444444445],[4.853333333333333,58.52777777777778],[4.669444444444445,58.89666666666667],[4.4991666666666665,59.60638888888889]]]]}},
+    { "type": "Feature", "properties": { "id": "EPDE", "prefix": [ "EPDE" ], "name": "Deblin Approach"}, "geometry": { "type": "MultiPolygon", "coordinates": [ [ [ [ 21.450833333333332, 51.611111111111114 ], [ 21.662222222222219, 51.4725 ], [ 22.159722222222221, 51.321666666666673 ], [ 22.3025, 51.334722222222226 ], [ 22.455555555555556, 51.366666666666667 ], [ 22.463055555555556, 51.659166666666664 ], [ 21.815, 51.838333333333338 ], [ 21.786666666666669, 51.804722222222217 ], [ 21.450833333333332, 51.611111111111114 ] ] ] ] } },
```

Comment:
Minor change, could you ensure the linting matches the repo style, and remove the unnecessary whitespace between brackets and braces? Since the rest of the repo uses that styling, it makes it much harder for us to review it at a glance.